PyInstaller/loader/rthooks/pyi_rth_qt5plugins.py

```diff
@@ -50,3 +50,9 @@
 
 # We set "qt5_plugins" as only one path for Qt5 plugins
 QCoreApplication.setLibraryPaths([os.path.abspath(d)])
+
+# Fix some Qt5 errors in linux
+if sys.platform.startswith('linux'):
+    os.environ['FONTCONFIG_FILE'] = '/etc/fonts/fonts.conf'  # Address missing font configuration issue
+    os.environ['FONTCONFIG_PATH'] = '/etc/fonts/'
+    os.environ['QT_XKB_CONFIG_ROOT'] = '/usr/share/X11/xkb'  # Address keyboard input issue
```

Comment:
I would expect this to be the defaults in Qt anyway. Why is this necessary?

Reply:
In ubuntu 16.04, using PySide2 from conda forge, it uses qt 5.6.2 from anaconda default channel.
Both fonts and keyboard input do not work in the packaged pyinstaller executable. I don't know about other platforms or qt versions, if it still is an issue. For reference check: https://tutel.me/c/programming/questions/42595685/this+application+failed+to+start+because+it+could+not+find+or+load+the+qt+platform+plugin+quotxcbquot+in+quotquot.

Reply:
Adding those solved the issue, i'd appreciate some feedback, on if its a version specific issue, and if that's the case, I can remove it.

Reply:
please refer to https://github.com/pyinstaller/pyinstaller/issues/3724